packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -67,18 +67,25 @@ class TextSpan extends InlineSpan {
   /// For the object to be useful, at least one of [text] or
   /// [children] should be set.
   const TextSpan({
-    this.text,
-    this.children,
+    String text,
+    List<InlineSpan> children,
     TextStyle style,
     this.recognizer,
     this.semanticsLabel,
-  }) : super(style: style,);
+  }) : _text = text,
+       _children = children,
+       super(style: style,);
 
-  /// The text contained in the span.
+  /// The text contained in this span.
   ///
   /// If both [text] and [children] are non-null, the text will precede the
   /// children.
-  final String text;
+  ///
+  /// This getter does not include the contents of its children.
+  @override
+  String get text => _text;
```

Comment:
This change and below should not be necessary. final fields are equivalent to a getter

Reply:
Ahh thanks, good to know!

Reply:
Will it override the getter without a @override?

Reply:
@override is an analyzer hint, but it is required for our lints

Reply:
... I think!